src/Sets/HalfSpace.jl

```diff
@@ -484,3 +477,14 @@ function translate(hs::HalfSpace{N}, v::AbstractVector{N}; share::Bool=false
     b = hs.b + dot(hs.a, v)
     return HalfSpace(a, b)
 end
+
+function _linear_map_hrep_helper(M::AbstractMatrix{N}, P::HalfSpace{N},
+                          algo::AbstractLinearMapAlgorithm) where {N<:Real}
+    constraints = _linear_map_hrep(M, P, algo)
+    if length(constraints) == 1
+        c = first(constraints)
+        return HalfSpace(c.a, c.b)
```

Comment:
Should `c` not be a `HalfSpace` already? :smile:

Reply:
i did this for clarity (and to ensure returning a new HalfSpace instance) but i think we can just return `fist(constraints)` directly :+1: